openverse_catalog/dags/templates/create_provider_ingester.py

```diff
@@ -0,0 +1,152 @@
+"""
```

Comment:
I think these (and the other template files) might be the result of merge conflicts, they'll need to be dropped before this is merged. (Incidentally this is also causing the weird testing failures we're seeing!)

Reply:
I'm missing something here. What would need to be dropped?

Reply:
OK, I tried to fix the test failures, and yeah, definitely did not succeed. I'm also getting errors about rawpixel api keys. Would love some advice!

Reply:
Oh, I mean to say these files should be deleted entirely 🙂 I'll go ahead and do that

Reply:
I don't know why this comment is in the wrong place. Please forgive me!